lib/components/Alert.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+
+/**
+ * Provides one of four standard alert types.
+ * An alert consists of a box with a light background color,
+ * an icon, a headline, and a short explaination.
+ *
+ * The alert type determines the background color and icon displayed
+ * 
+ * Alert types are 'success', 'warning', 'error', 'info'
+ * The type defaults to 'info' if no type is sent via props
+ *
+ * If the type is 'error' and no role is specified, role defaults to 'alert'
+ *
+ * @param {string} type  Sets the alert type
+ * @param {string, required} title Text for the headline
+ * @param {ndoe, required} body  Text for the description
+ * @param {string} role  ARIA role type
+ */
+export default function Alert ({type, title, body, role}) {
```

Comment:
Ultimately I will defer to you, but I want to push back a little here. I think stateless functional components are the way to go if there is no need for state or lifecycle hooks. Even though, for the moment, there's little optimization gain, it makes it super obvious at a glance that it's a dumb presentational component. This guy [sums up the arguments pretty well](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc#.soc5jxwms).  Prop types are enforced with console errors just like classes ... unless you were referring to something else(?).